packages/table/src/cell/formats/truncatedFormat.tsx

```diff
@@ -50,21 +59,29 @@ export interface ITruncatedFormatProps extends IProps {
     truncationSuffix?: string;
 }
 
-@PureRender
-export class TruncatedFormat extends React.Component<ITruncatedFormatProps, {}> {
+export interface ITruncatedFormatState {
+    truncated: boolean;
```

Comment:
i might call this guy `isTruncated` so it's obviously a boolean. would read a little better.